rest/permissions.py

```diff
@@ -105,7 +106,7 @@ def has_permission(self, request, view):
         except PermissionDenied:
             return False
 
-        if request.method in SAFE_METHODS:
+        if request.method in self.SAFE_METHODS_CUSTOM:
```

Comment:
What you need (if you did not already implemented it) is a way for the interface to allow edit according to the legged user:
- if the user has permissions to edit taxonomy, the interface should allow the edit (and the REST APIs to edit taxonomy should work for him, but they already do it)
- if the user does not have permissions to edit taxonomy the interface should not allow even to show the possibility to edit